rust/opendp_tooling/src/codegen/r/r.rs

```diff
@@ -229,7 +230,7 @@ fn generate_doc_block(
     };
 
     format!(
-        r#"{description}
+        r#"{flag_doc}{description}
```

Comment:
I'd rather not make the requirements the first line in the doc. In particular, in R, the first line is the only thing that shows in the reference index: https://docs.opendp.org/en/stable/api/r/reference/index.html (a lot of these could be better anyways, but that's separate).

What if you instead add the set of required features in `BootstrapDocstring::from_attrs`? You might need to pass the list of features into that function. Then you wouldn't need lang-specific codegen and could choose where it gets placed in the resulting docstring.

Reply:
> you wouldn't need lang-specific codegen

Though both the Python and R messages are `enable_features("some_feature")` right now, this could be different between target languages. (Even with Python, initially I had `dp.enable_features("some_feature")`.) But maybe I'm misunderstanding your concern?

Moving this off the first line does make sense: Let me see about the end of the description.